CorsixTH/Src/th_map.cpp

```diff
@@ -645,6 +629,26 @@ void THMap::setParcelOwner(int iParcelId, int iOwner)
     _updatePurchaseMatrix();
 }
 
+void THMap::_checkDividers(THMapNode* pNode, const THMapNode* pOriginalNode,
```

Comment:
The only access this needs to private THMap properties is m_pPlotOwner and there is a public method for getting the plot owner.

In conformance with C++ Core Guidelines C.4, this method should not be a part of the THMap class. Since it's not used outside of this file, it can be defined as 
```cpp
static void check_dividers(THMapNode* pNode ...)
{
...
}
```